gradle.lockfile

```diff
@@ -1,107 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```

Comment:
@CianoDanilo why delete entire file?

Reply:
File added again in [fbb18b0](https://github.com/pagopa/pagopa-payment-wallet-cdc-service/pull/6/commits/fbb18b0538277f2cfe5fa46e848f7e4a39b73e51)